### PR TITLE
new implementation for collaboration tooltip

### LINKED ISSUE #8364: Implement new tooltip for collaboration.
collaboration tooltip1

collaboration tooltip2

We would like to see new tooltip modal when collaboration is started (image 1) and later on clicked the collaboration link to copy to clipboard (image 2).

## HINT
**Basic CSS attributes for Image 1:**
width: 250px, height: 180px, background: #FFFFFF, box-shadow: 0px 2px 10px 0px rgba(0,0,0,0.05), border-radius: 4px

font-size: 14px;
letter-spacing: 0.13px;
line-height: 20px;

color: #48BA7D; (green) for header of image 1
color: #727272;

**Basic CSS attributes for Image 2:**

width: 250px, height: 125px, background: #FFFFFF, box-shadow: 0px 2px 10px 0px rgba(0,0,0,0.05), border-radius: 4px; 

font-size: 14px;
color: #727272;
letter-spacing: 0.11px;
line-height: 20px;

## Acceptance Criteria

The pull request must have screenshots that proves new tooltip is implemented. 

<bountysource-plugin>
---
There is a **[$60 open bounty](https://www.bountysource.com/issues/37843755-implement-new-tooltip-for-collaboration?utm_campaign=plugin&utm_content=tracker%2F41989991&utm_medium=issues&utm_source=github)** on this issue. Add to the bounty at [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F41989991&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by cansirin**
## Description

This PR changes tooltip style for collaboration messages.
## Motivation and Context

Fixes #8364 
## How Has This Been Tested?

Manually.
## Screenshots (if appropriate):

image
## Types of changes
- [x] New feature (non-breaking change which adds functionality)
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by usirin** [comment]
Text color on the 2nd tooltip (the one pops up on click) is too dark, needs to be adjusted.

File: client/app/lib/styl/tooltip.styl [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  line-height                 20px
+
+  div.wrapper
+    bg                        color, white
+    shadow                    0 10px 20px 5px rgba(0,0,0,0.15)
+    padding                   20px 20px 20px 20px
+
+  .copied-link strong
+    font-weight               600
+    color                     $lightColor
```
Comment (usirin) [inline client/app/lib/styl/tooltip.styl]: this color probably needs to be applied to only `.copied-link`, only `font-weight` needs to be applied to `strong` element.

**Review by usirin — CHANGES_REQUESTED** [review]

File: client/app/lib/styl/tooltip.styl (hunk 1 above)
Comment (cansirin) [inline client/app/lib/styl/tooltip.styl]: image

File: client/app/lib/styl/tooltip.styl [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  &.just-text
+
+    div.wrapper
+      shadow                  0 3px 7px rgba(255, 255, 255, 0.25)
+      bg                      color, white
+      color                   #727272
+
+      font-size               12px
+      font-weight             300
```
Comment (sinan) [inline client/app/lib/styl/tooltip.styl]: Once terminal is opened, this is gonna affect all the tooltips in the app. We should namespace this, or make sure every other tooltip looks good!

**Review by sinan — CHANGES_REQUESTED** [review]

File: client/ide/lib/views/statusbar/idestatusbar.coffee [hunk 3]
```diff
@@ -44,15 +44,29 @@ module.exports = class IDEStatusBar extends kd.View
...
+      partial: ''' <h3>Collaboration session is started. Share link to invite.</h3>
+      <p>This is your collaboration link. You can share this link anytime to invite someone to your
+      session. Click link to copy!</p>
+      '''
+
+    copyLink = new kd.CustomHTMLView
+      cssClass: 'copied-link'
+      partial: '''
+        <strong>Collaboration link is copied to clipboard.</strong>
+        Now, you can share it to invite someone to your session.
```
Comment (sinan) [inline client/ide/lib/views/statusbar/idestatusbar.coffee]: you can drop the second sentence here.

File: client/app/lib/styl/tooltip.styl (hunk 2 above)
Comment (usirin) [inline client/app/lib/styl/tooltip.styl]: https://github.com/koding/koding/pull/9153/files#diff-2d9ea1bbdb9995560f3945091858530fR4

It's already namespaced.

**Review by usirin — APPROVED** [review]

**Comment by usirin** [comment]
@sinan My requested changes are done, if you are good we can merge this one.

File: client/app/lib/styl/tooltip.styl (hunk 2 above)
Comment (sinan) [inline client/app/lib/styl/tooltip.styl]: you mean `.is-light`

**Review by sinan — APPROVED** [review]

**Pull request merged**